From afa16de5c385619e73ea63fe7a622e6bf40de985 Mon Sep 17 00:00:00 2001
From: arathinx <ayyappanx.rathinam@intel.com>
Date: Wed, 20 Nov 2019 10:33:31 +0800
Subject: [PATCH] PONRTSYS-5485: Do not deactivate clocks needed by TEP in IAP

Resolution: Bypass the default disabling for SCPU (4kec MIPS core) & SECPT(EIP123) clks
---
 drivers/clk/intel/clk-prx300.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/clk/intel/clk-prx300.c b/drivers/clk/intel/clk-prx300.c
index 856b71dd2577..f64e84dc131a 100644
--- a/drivers/clk/intel/clk-prx300.c
+++ b/drivers/clk/intel/clk-prx300.c
@@ -176,10 +176,10 @@ static const struct intel_clk_branch prx300_branch_clks[] __initconst = {
 		   17, GATE_CLK_HW, 0),
 	INTEL_GATE(PRX300_GCLK_URT1, "g_uart1", NULL, 0, CGU_GATE1,
 		   18, GATE_CLK_HW, 0),
-	INTEL_GATE(PRX300_GCLK_SECPT, "g_secpt", NULL, 0,   CGU_GATE1,
-		   21, GATE_CLK_HW, 0),
-	INTEL_GATE(PRX300_GCLK_SCPU, "g_scpu", NULL, 0, CGU_GATE1,
-		   22, GATE_CLK_HW, 0),
+	INTEL_GATE(PRX300_GCLK_SECPT, "g_secpt", NULL, CLK_IGNORE_UNUSED,
+		   CGU_GATE1, 21, GATE_CLK_HW, 0),
+	INTEL_GATE(PRX300_GCLK_SCPU, "g_scpu", NULL, CLK_IGNORE_UNUSED,
+		   CGU_GATE1, 22, GATE_CLK_HW, 0),
 	INTEL_GATE(PRX300_GCLK_MPE, "g_mpe", NULL, 0, CGU_GATE1,
 		   23, GATE_CLK_HW, 0),
 	INTEL_GATE(PRX300_GCLK_TDM, "g_tdm", NULL, 0, CGU_GATE1,
